frontend/src/main/pages/HomePage.js

```diff
@@ -47,6 +47,8 @@ export default function HomePage() {
       }
     }, [currentUser]
   );
+
+  const firstName = currentUser?.root ? currentUser?.root?.user?.givenName : "";
```

Comment:
If root exists but not givenName for any reason, this will display something along the lines of `Howdy Farmer null`. Better solution here is to drop the ternary and use `||` fallback instead..

Reply:
Agreed with Chris.

Suggest that you do a few things:
1.  Fix this along the lines that Chris suggests, i.e. something like 
```suggestion
  const firstName = (currentUser?.root?.user?.givenName) || "";
```

2. Add `msw` support into the HomePage.story.js file 
3. Set up multiple stories that have fixtures with (a) currentUser with a name present (b) no currentUser at all (c) currentUser is present, but root.user.givenName is not present.